app/views/reverse_dependencies/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <% @title = t('.title', name: @rubygem.name) %>
-<% @subtitle = @latest_version&.slug %>
+<% @subtitle =  t(".subtitle", name: @rubygem.name) %>
```

Comment:
Can we add a line break between the title so this just looks nice on laptop screens.

Reply:
These variables are part of the layout, so any changes here will be done on all pages.

Reply:
🤦

Reply:
I agree with the long subtitle text this looks non-pleasing. Maybe we can a new variable in layout which uses linebreak?
```
if @newline_subtitle
   tag on new line
```